eng/pipelines/templates/jobs/archetype-sdk-client.yml

```
@@ -4,6 +4,7 @@ parameters:
   Coverage: ''
   CtestRegex: .*
   BuildReleaseArtifacts: true
+  CoverageReportPath: 'sdk/*/*/*cov_xml.xml'
```

Should we put coverage files into a coverage folder?
```suggestion
  CoverageReportPath: 'sdk/*/*/*/cov/*.xml'
```

---

I don't think so.  I don't see any benefit :P.
Do we put output binaries into one folder?  or any other output structure?  Nop, 
End user will likely never generate xml reports, for end-user, we offer the HTML report which is created in its own folder